container/Containerfile

```diff
@@ -13,8 +13,11 @@ FROM alpine
 RUN apk add python3
 RUN addgroup --system ovirtimg
 RUN adduser --system --ingroup ovirtimg ovirtimg
-RUN mkdir /run/ovirt-imageio /etc/ovirt-imageio
+RUN mkdir /run/ovirt-imageio /etc/ovirt-imageio /images
 COPY --from=build /venv /venv
 COPY conf.d /etc/ovirt-imageio/conf.d
+COPY file.json file.json
+COPY disk.img /images
+RUN chown ovirtimg:ovirtimg /images/disk.img
 EXPOSE 80/tcp
-CMD /venv/bin/ovirt-imageio
+CMD /venv/bin/ovirt-imageio -t file.json
```

Comment:
The name of the ticket and its location should be set by the user of the container. One way to pass the info is an environment variable.

Reply:
The location could be pre-defined and let the user pick only the name through an env variable.

So we have:
```Containerfile
ARG TICKET_NAME # or ARG TICKET_NAME=ticket.json (for default value)
VOLUME [ "ticket" ]
CMD /venv/bin/ovirt-imageio -t /ticket/${TICKET_NAME}
```

And it can be run by doing:
```
TICKET_NAME=ticket.json podman run \
    --env-host \
    --rm \
    --privileged \
    -it -p 8080:80 \
    -v ./path/to/ticket:/ticket
    -v ./path/to/images/:/path/of/the/ticket/img \
    localhost/ovirt-imageio:latest
```

Otherwise, we have to set the whole path of the file in an env variable, or have two, one for the folder and one for the name. And we only need one ticket every time it is run, so I find it easier to just assume a folder that will contain the ticket, and let the user specify which ticket file to load.

There is not much difference, I tried both, and it works either way. Just feels easier for users if we assume a mount point in the container for tickets. Also, I don't think it is interesting to try different locations for tickets (it may be more interesting for images), so shouldn't be a problem to have it pre-defined.

Reply:
I'm not sure why we need this:

    -v ./path/to/images/:/path/of/the/ticket/img

Instead of

    -v /path/to/image:/disk:Z

But it is true that using well known path for the ticket makes it easier to use. For the image I'm not sure we can require certain path since in kubvirt they always use /disk.img and we want to be compatible with this. On the other hand we don't want to require this location.

For k8s the ticket is likely to be passed using configmap, and in this case the configmap keys are converted to mounts inside the container, but I'm not sure how it looks inside the container.

In summary, we want to have a flexible solution that can work for any use case. Making it easier to use for some cases is an optimization that can be done later.

Reply:
Just for the record, I was just talking on defining the ticket path, the image path I agree that should remain flexible, and can be determined by the information in the ticket and the `--volume` specified at runtime.

This was just an example:
```
-v ./path/to/images/:/path/of/the/ticket/img
```
I wanted to put the focus on the ticket volume. Your counterexample looks good to me too.